src/test/java/com/quizapp/enterprise/EnterpriseApplicationTests.java

```diff
@@ -35,16 +38,17 @@ void verifyGameProperties() {
 
     /**
      * Validate that the GameService can save and return games. (based on individual assignment)
+     *
      * @author Mahesh Gowda
      */
     @Test
     void verifyAddAndRemoveGames() {
-        String gameCode =  "FIRST";
+        String gameCode = "FIRST";
 
         Game game = new Game();
         game.setGameCode(gameCode);
 
-        List<Game> games = GameService.getAllGames();
+        List<Game> games = gameService.getAllGames();
```

Comment:
Using the autowired instance to fix this test because you can't call this method off the class unless it is static